### PR TITLE
Initial documentation for IMQ Alerts. Fixes IMQ API reference. Do not merge please.

### PR DESCRIPTION
**Pull request by featalion**

### CODE REVIEW

File: mq/reference/api/index.md [hunk 1]
```diff
@@ -723,7 +896,7 @@ Get a message by ID.
 ### Endpoint
 
 <div class="grey-box">
-GET /projects/<span class="variable project_id">{Project ID}</span>/queues/<span class="variable queue_name">{Queue Name}</span>/messages/peek
```
Comment (treeder) [inline mq/reference/api/index.md:726]: Why is this removed?

Comment (featalion) [inline mq/reference/api/index.md:726]: It was not removed, it was changed. Endpoint for "Get Message by ID" is different, see line below the comment.
